docs/reference/commandline/create.md

```diff
@@ -81,6 +81,7 @@ Creates a new container.
       --security-opt=[]             Security options
       --stop-signal="SIGTERM"       Signal to stop a container
       --shm-size=[]                 Size of `/dev/shm`. The format is `<number><unit>`. `number` must be greater than `0`.  Unit is optional and can be `b` (bytes), `k` (kilobytes), `m` (megabytes), or `g` (gigabytes). If you omit the unit, the system uses bytes. If you omit the size entirely, the system uses `64m`.
+      --storage-opt=[]		    Set storage driver options per container
```

Comment:
Fixed. I added the description in the `Examples` section below. Let me know if you want me to put it somewhere else.